fastlane/lib/fastlane/setup/crashlytics_beta.rb

```diff
@@ -64,7 +62,7 @@ def lane_template
              build_secret: '#{@beta_info.build_secret}',#{crashlytics_path_arg}
                    emails: ['#{@beta_info.emails.join("', '")}'], # You can list more emails here
                  # groups: ['group_alias_1', 'group_alias_2'], # You can define groups on the web and reference them here
-                    notes: 'Distributed with fastlane 🚀',
```

Comment:
Yeah, our tester let us know that it wasn't making it through the Beta system correctly 😢